org.civicrm.reqangular/src/common/decorators/angular-date/datepicker-popup.js

```diff
@@ -16,22 +16,30 @@ define([], function () {
                  */
                 attrs.datepickerPopup = HR_settings.DATE_FORMAT;
 
-                scope.$watch('isOpen', function (value) {
+                /**
+                 * Calculates position of the datepicker popup.
+                 * @param value
+                 */
+                var adjustPopupPosition = function (value) {
```

Comment:
I'm using `scope` and `element` vars inside this function. If I move this function outside compile closure, these variables will become undefined.